dockstore-webservice/src/main/java/io/dockstore/webservice/core/Entry.java

```diff
@@ -129,20 +129,28 @@
     @Column
     @ApiModelProperty(value = "This is the email of the git organization", position = 6)
     private String email;
+
     @Column
     @ApiModelProperty(value = "This is the default version of the entry", position = 7)
     private String defaultVersion;
+
     @Column
     @JsonProperty("is_published")
     @ApiModelProperty(value = "Implementation specific visibility in this web service", position = 8)
     private boolean isPublished;
 
     @Column
-    @ApiModelProperty(value = "Implementation specific timestamp for last modified", position = 9)
+    @ApiModelProperty(value = "Implementation specific timestamp for last modified."
```

Comment:
Given the way you are concatenating, how does this appear in both the database and in the generated Swagger/OpenAPI? It seems like there may not be be a space after the period, e.g., you might get `...last modified.Tools->...

If it is an issue, this comment applies to almost all of your changes.

Update: I see in the generated files that there is no space after the period.